### DISCUSSION TITLE
Local repo guide

### DISCUSSION THREAD

**Discussion by muzzol** [semaphoreui/semaphore#892] [Q&A]
Is there a guide to create a local repository?
Right now docs are empty and I can't make it work, so I'd really appreciate a step by step guide to setup local directory repo.

Also, I can see some unanswered issues/questions about local repo. I this something devs are working on making it easier or git is the only planed route?

**Comment by ghost** [comment]
I read about it on #532 but the little guide there was insufficient. at least for me. I pointed semaphore to two different repos, one in tmp and one in my personal user, but neither worked.

**Reply by ghost** [reply]
https://issueantenna.com/repo/ansible-semaphore/semaphore/issues/931 
this isnt me, but my problem is identical.
